### PR TITLE
Add CRA banners to security and certs pages

### PR DESCRIPTION
**Pull request by britneywwc**
_Go live date June 3rd_
## Done

- Add CRA banners to `/security` and `/security/certification` pages

## QA
- Go to demos for `/security`, check that changes are updated as per [copy docs](https://docs.google.com/document/d/13RcVvHBAoywfk3o5Dy-16F3KrMytL9-2v6wWbW444Ko/edit)
- Go to demos for `/security/certification`, check that changes are updated as per [copy docs](https://docs.google.com/document/d/1yEocR1WXQvN_B1L1yBYrG_0D7ikS6QhcOz27sYs-teI/edit)

## Issue / Card

Fixes [WD-11271](https://warthogs.atlassian.net/browse/WD-11271)

## Screenshots

[If relevant, please include a screenshot.]

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

[WD-11271]: https://warthogs.atlassian.net/browse/WD-11271?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Thanks @britneywwc, few points below:
1. Can you use the [information notification pattern](https://vanillaframework.io/docs/patterns/notification#severity-levels) please? LIke we do on the homepage. 
2. Can you make the notification an include, so if we change it it will change in both places.

**Comment by britneywwc** [comment]
Thanks for the review @anthonydillon , I've applied your suggested changes. 
Let me know if there's anything else I should address.

File: templates/shared/_cra-banner.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<section class="p-strip">
+  <div class="row p-notification--information">
+    <div class="p-notification__content">
+      <div class="p-notification__message">Need information about the CRA? Canonical is committed to delivering Cyber Resilience Act (CRA) compliant Ubuntu. To learn more, 
+        <a href="#get-in-touch">contact our sales team.</a>
```
Comment (anthonydillon) [inline templates/shared/_cra-banner.html:5]: Sorry should have seen this. The link should be to a static page like "/security/contact-us" with the class which invokes the modal.

**Review by anthonydillon — CHANGES_REQUESTED** [review]

File: templates/shared/_cra-banner.html (hunk 1 above)
Comment (britneywwc) [reply templates/shared/_cra-banner.html:5]: The links on both copy docs seem to be "#get-in-touch" attached to the original link. Should I change it to your suggestion with the the static page link instead?

Comment (anthonydillon) [reply templates/shared/_cra-banner.html:5]: You could do but the authors do not really know how the system works.

**Comment by anthonydillon** [comment]
Should there be such a large diff?

**Comment by britneywwc** [comment]
> Should there be such a large diff?

The large diff is mostly from the file linting

File: templates/shared/_cra-banner.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<section class="p-strip">
```
Comment (anthonydillon) [inline templates/shared/_cra-banner.html:1]:
```suggestion
<section class="p-strip u-no-padding--bottom">
```
Personally I would add no padding to the bottom as there is a large gap between the notice and the following content. This also aligns with the usage on the homepage.

**Review by anthonydillon — APPROVED** [review]
This looks good with a small suggestion in the code.

**Pull request merged**